packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -147,10 +148,10 @@ const ComboBox = (props) => {
   };
 
   useEffect(() => {
-    if (onInputChange) {
-      onInputChange(inputValue);
+    if (handleInputChange) {
+      handleInputChange(inputValue);
     }
-  });
+  }, [handleInputChange, inputValue]);
```

Comment:
```suggestion
  }, [inputValue]);
```

Reply:
I was still getting the lint error about dependencies here when I removed it earlier but it might be a false positive